aya/src/obj/mod.rs

```diff
@@ -657,10 +667,19 @@ mod tests {
 
     #[test]
     fn test_parse_map_def() {
+        #![allow(unused_variables)] // map_def is used by the assertion
         assert!(matches!(
-            parse_map_def("foo", &[]),
+            parse_map_def("foo", &[0u8; std::mem::size_of::<bpf_map_def>() + 10]),
```

Comment:
The old test is still valid, please revert this change

Reply:
Reverted in 1404d12a7e29ab02225bc13fd36794f6c6206db1